### ISSUE TITLE
Telemetry on shield click (NEEDS TRIAGE)

### ISSUE THREAD

**Issue by locklin** [brave/browser-laptop#7593]
I am assuming this means clicking on the lion phones home. Not clear if the telemetry includes website information from the 4 word postit.

**Comment by bsclifton** [comment]
@bradleyrichter @aekeus is this information we'd like to collect (if people are opted-in to sharing anonymous telemetry data)?

**Comment by bradleyrichter** [comment]
Yes. We need to do anonymous crowd-sourced shield logging and use the data to pre-set shields for known breakage.

**Comment by aekeus** [comment]
Agreed. This needs to be done using statistical aggregation using something like the RAPPOR method from Google. https://github.com/google/rappor
